<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMAMSGroupSettingsWidget</class>
 <widget class="QWidget" name="QMAMSGroupSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Gruppen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="tvGroup">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pbAddGroup">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Hinzufügen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemoveGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAccessMode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Allgemeine Rechtzuweisung zur Gruppe</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Zugewiesene Rechte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvGroupAccessMode">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbAddAccessMode">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRemoveAccessMode">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Löschen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Verfügbare Rechte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvAccessMode">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>206</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbAddAccessMode</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSGroupSettingsWidget</receiver>
   <slot>addAccessMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSGroupSettingsWidget</receiver>
   <slot>addGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRemoveAccessMode</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSGroupSettingsWidget</receiver>
   <slot>removeAccessMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRemoveGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSGroupSettingsWidget</receiver>
   <slot>removeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addGroup()</slot>
  <slot>removeGroup()</slot>
  <slot>addAccessMode()</slot>
  <slot>removeAccessMode()</slot>
 </slots>
</ui>
